src/utils/whatsapp-link-generator.ts

```diff
@@ -1,20 +1,21 @@
 
-export const generateWhatsappLink = (contactName: string, productName: string, couponCode: string,couponPercentageValue:string, phone: string, controlledMessage?: string) => {
-    let message = controlledMessage || generateWhatsappUpsellMessage(contactName, productName, couponCode,couponPercentageValue);
+export const generateWhatsappLink = (contactName: string, productName: string, phone: string, controlledMessage?: string) => {
+    let message = controlledMessage || generateWhatsappUpsellMessage(contactName, productName);
     const whatsappLink = `https://wa.me/${phone}?text=${encodeURIComponent(message)}`;
-    console.log(whatsappLink)
+    console.log(whatsappLink);
     return whatsappLink
 }
 
 const generateProductLink = (productName: string) => {
     const productRoute = productName.split(' ').join('-').toLowerCase();
+    // This is a sample link, you should replace it with your own product page link
     return `https://etaybarzilay.wixstudio.io/order-custom-actions/product-page/${productRoute}`;
```

Comment:
Maybe worth extracting the const part of the url to a variable which will be easier to replace